sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -116,7 +120,7 @@ extern "C" {
 typedef struct {
     uint8_t *data;  /**< pointer to the beginning of the buffer holding data */
     uint8_t size;   /**< size of the buffer */
-    uint8_t index;  /**< current inxed in the buffer */
+    uint8_t index;  /**< current index in the buffer */
```

Comment:
when this is rebased, please keep this unrelated change in a separate commit

Reply:
Sure.